src/components/img.ts

```diff
@@ -38,7 +38,7 @@ const Img: FC<Props> = ({ image, sizes, className }) =>
         styledH('img', {
             src: image.src,
             alt: image.alt.withDefault(''),
-            className: 'js-launch-slideshow',
+            className: image.launchSlideshow ? 'js-launch-slideshow' : null,
```

Comment:
I have a vague memory that this might appear in the DOM with the attribute `"null"` or `""` or something, but I may be completely wrong in which case please ignore 🤔